### DISCUSSION TITLE
methode DHT22 in dht module don't work

### DISCUSSION THREAD

**Discussion by Francois25** [russhughes/st7789_mpy#119] [Q&A]
Hi,
First, thank for your job.

I use your ST7789 firmware because I need to use urequest module and dht module.
But the dht module don't work, maybe you can help me.

First, when I import the module in ma code with the simple command 'import dht' I have this error message :

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "dht.py", line 1, in <module>
ImportError: can't import name dht_readinto

I need to code a try-exept for that it work.

try:
    import dht
except:
    import dht

Second, when import dht is ok, I create a variable to assignate the pin of my ESP32 board like that :

`sensor = dht.DHT22(machine.Pin(2))`

and I have another error message:

AttributeError: 'module' object has no attribute 'DHT22'

Can you share me the dht.py file check wy it don't work or maybe you can change the firmware for resolve this mistake.

Have a nice day.
Best regards.
François

**Comment by russhughes** [comment]
I'll look into this.

**Reply by Francois25** [reply]
I've not precise but I use the T-DISPLAY-ESP32 firmware

**Comment by demoontz** [comment]
I have the same issue with the same firmware.

**Comment by russhughes** [comment]
I'll look into this tonight.

**Comment by dusan-zervan** [comment]
Hello, SPI Baudrates above 26.7 MHz are not working again in this release? (T-Display) I have to downgrade to 1.19.1-738: https://github.com/russhughes/st7789_mpy/blob/50f91804ddc5016a1143d3b274559cae778ba338/firmware/T-DISPLAY-ESP32/firmware.bin

**Reply by russhughes** [reply]
Should be fixed.
